apps/website/src/components/layout/navbar/DesktopMenu.tsx

```diff
@@ -73,6 +73,9 @@ export function DesktopMenu() {
     <div className="hidden grow flex-col gap-2 lg:flex">
       <div className="flex items-center justify-end gap-2">
         <ul className="contents">
+          <li>
+            <ThemeToggleButton />
+          </li>
```

Comment:
I think this should probably be grouped with the notifications button, not as part of the social links